sessions/recap-project-2/quiz-app/README.md

```diff
@@ -108,7 +130,7 @@ users should be informed about the amount of characters left.
   result
 - Think of ways to use the same logic for both form fields and to not repeat your code
 
-## 5. Bonus
+## Bonus
```

Comment:
```suggestion
### Bonus: More Interactivity
```

Reply:
It was decided to keep the Bonus as h2.
Task are mandatory feature, meanwhile bonus and bonus, extra